config/routes.rb

```diff
@@ -153,7 +150,7 @@
     # saved searches
     scope '/user', controller: 'users' do
       get :unread
-      get 'tracking/:article_id' => 'users#tracking', :as => :readraptor
+      get 'tracking/:article_id' => 'users#tracking', as: :readraptor
 
       resources :saved_searches, only: [:create, :destroy]
 
```

Comment:
Line is too long. [95/80]<br>Unnecessary spacing detected.<br>Put one space between the method name and the first argument.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.